packages/flutter/lib/src/widgets/app.dart

```diff
@@ -1197,7 +1198,29 @@ class WidgetsApp extends StatefulWidget {
   /// and view what widgets and render objects associated with it. An outline of
   /// the selected widget and some summary information is shown on device and
   /// more detailed information is shown in the IDE or DevTools.
-  static bool debugShowWidgetInspectorOverride = false;
+  @Deprecated(
+    'Use debugShowWidgetInspectorOverrideNotifier.value instead. '
+    'This feature was deprecated after v3.13.0-19.0.pre.'
+  )
+  bool get debugShowWidgetInspectorOverride =>
+      debugShowWidgetInspectorOverrideNotifier.value;
+  @Deprecated(
+    'Use debugShowWidgetInspectorOverrideNotifier.value instead. '
+    'This feature was deprecated after v3.13.0-19.0.pre.'
+  )
+  set debugShowWidgetInspectorOverride(bool value) {
+    debugShowWidgetInspectorOverrideNotifier.value = value;
+  }
+
+  /// If true, the WidgetInspector is added to the tree and made visible.
+  ///
+  /// Used by the `debugShowWidgetInspector` debugging extension.
+  ///
+  /// The inspector allows you to select a location on your device or emulator
+  /// and view what widgets and render objects associated with it. An outline of
+  /// the selected widget and some summary information is shown on device and
+  /// more detailed information is shown in the IDE or DevTools.
+  static ValueNotifier<bool> debugShowWidgetInspectorOverrideNotifier = ValueNotifier<bool>(false);
```

Comment:
Should this be

```suggestion
  static ValueListenable<bool> debugShowWidgetInspectorOverrideNotifier = ValueNotifier<bool>(false);
```

The fact that it is implemented as a ValueNotifier is likely an implementation detail?

Reply:
This is the one from the Flutter code base: https://master-api.flutter.dev/flutter/foundation/ValueListenable-class.html

It should be accessible here?

Reply:
I see that you removed this from the public interface, so this comment doesn't really matter anymore :)